commands.h

```diff
@@ -1,83 +1,43 @@
-#ifndef COMMANDS_H
-#define COMMANDS_H
-#include <stdbool.h>
+#define G_CMD(c) bool cg_##c(arg_t); static const cmd_t g_##c = { cg_##c, MODE_ALL };
+#define I_CMD(c) bool ci_##c(arg_t); static const cmd_t i_##c = { ci_##c, MODE_IMAGE };
+#define T_CMD(c) bool ct_##c(arg_t); static const cmd_t t_##c = { ct_##c, MODE_THUMB };
```

Comment:
Voting to not specify the argument type so we don't have to pass dummy values to those functions that don't take an argument

Reply:
But all the functions in keybinding _has_ to take arg_t as argument. So not specifying it doesn't make much sense and just creates some confusion.

Reply:
Forgot that the bound functions had to be of a certain type. At that point, it just comes down to how they are manually called. And the only time they are manually called is for quit which doesn't have an argument and I find specifying one confusing.

Like `quit(None)` == `quit(0)`, which may lead someone to think that the argument is specifying the exit status like `exit` does. And would be surprised that `quit(1)` behaves just as `quit(0)`.

I'm focusing on cg_quit because that's the only case where the difference matters in master. Of course if we made quit use its argument, I'd lose my objection.

Reply:
> Like quit(None) == quit(0), which may lead someone to think that the argument is specifying the exit status like exit does. And would be surprised that quit(1) behaves just as quit(0).

I agree, which is why I changed `cg_quit(0)` to `cg_quit(None)`.

> if we made quit use its argument

I've thought about it. But out of scope for this PR.

Reply:
>I've thought about it. But out of scope for this PR.

I feel like half the things done are "out of scope" going by the title. Anyways, that's the only thing holding up my approval, but feel free to go ahead if some else approves.